app/controllers/order_items_controller.rb

```diff
@@ -0,0 +1,82 @@
+class OrderItemsController < ApplicationController
+  before_action :find_item, except: :add
+
+  # # I think this is leftover from the conversion of adding a cart item to the cart to the products controller.
+  # def add
+  #   item = OrderItem.create(create_params)
+  #
+  #   flash[:error] = item.errors.messages if item.errors.messages
+  #
+  #   redirect_to root
+  # end
+
+  def more # of a particular item in a cart; increases the quantity of an item in the cart
+    stock = @item.product.stock
+    current_quantity = @item.quantity_ordered
+
+    if current_quantity < stock
+      orders = Order.pending
+
+      # this belongs in the model
```

Comment:
I agree with this comment. I think this method is doing way too much work, and it's not immediately clear to me what it's doing or why. It's worth spending some time here talking about how to simplify the process. What information do you need? How can you accomplish it in fewer iterations? How many queries are truly necessary? Let me know if you'd like me to pair on this with you.